src/test/java/stepDefinition/HistorySteps.java

```diff
@@ -0,0 +1,81 @@
+package stepDefinition;
+
+import cucumber.api.DataTable;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+import org.junit.Assert;
+import org.openqa.selenium.support.ui.Select;
+import pageObjects.login.*;
+import utility.GetPropertiesValue;
+import utility.ScenarioContext;
+import utility.TestBase;
+
+import java.util.List;
+import java.util.Map;
+
+public class HistorySteps extends TestBase {
+    private GetPropertiesValue getPropertiesValue = new GetPropertiesValue();
+
+    public HistorySteps(ScenarioContext context) {
+        scenarioContext = context;
+    }
+
+    @Given("^an user login with account \"([^\"]*)\" and password as \"([^\"]*)\"$")
+    public void userLoginWithAccountAndPasswordAs(String username, String password) {
+        driver.get(getPropertiesValue.getPropValues("Base_URL.properties", "BASE_URL") + "/profile.php#login");
+        LoginPage.username().sendKeys(username);
+        LoginPage.password().sendKeys(password);
+        LoginPage.loginButton().click();
+    }
+    @When("^User input data as below$")
+    public void userInputDataAsBelow(DataTable table) throws InterruptedException {
+        List<Map<String, String>> tableInput = table.asMaps(String.class, String.class);
+        int i;
+        for (i = 0; i < tableInput.size(); i++) {
```

Comment:
xoá dòng  `int i;`, e có thể viết như này
for (int i = 0; i < tableInput.size(); i++) {